test/test_utils.py

```diff
@@ -202,9 +202,30 @@ def test_munge_cmdstan_versions(self):
             os.makedirs(os.path.join(tdir, 'cmdstan-2.22.0'))
             self.assertEqual(get_latest_cmdstan(tdir), 'cmdstan-2.22.0')
 
-    def test_cmdstan_version_at(self):
+    def test_cmdstan_version_before(self):
```

Comment:
This isn't really testing the utility, it's testing the failure conditions, so we might want a different name